python/mlc_chat/compiler/model/llama_quantization.py

```diff
@@ -39,3 +39,19 @@ def awq_quant(
         "",
     )
     return model, quant_map
+
+
+def no_quant(
+    model_config: LlamaConfig,
+    quantization: NoQuantize,
+) -> Tuple[nn.Module, QuantizeMapping]:
+    """Quantize a Llama2 model without quantization."""
+    model: nn.Module = LlamaForCasualLM(model_config)
+    model.to(quantization.model_dtype)
+    quant_map = QuantizeMapping({}, {})
+    model = quantization.quantize_model(
+        model,
+        quant_map,
+        "",
+    )
```

Comment:
Is it possible to simply skip `quantization.quantize_model` and use an empty `QuantizeMapping`?

Reply:
yes, removed.